migrations/20190403113644-notification-composite-index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = {
+  up: async function(queryInterface, DataTypes) {
+    await queryInterface.addIndex('Notifications', ['channel', 'type', 'webhookUrl', 'CollectiveId'], {
```

Comment:
We don't have enough database constraints in our current schema. This index is much appreciated!